### PR TITLE
Digi  6  create menu item

### PR DESCRIPTION
**Pull request by praveen957**

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj [hunk 1]
```diff
@@ -9,9 +9,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>DigiOrderEnterpriseModel</RootNamespace>
     <AssemblyName>DigiOrderEnterpriseModel</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:12]: add frame work as 4.7.2 only

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj [hunk 2]
```diff
@@ -9,9 +9,10 @@
...
     <FileAlignment>512</FileAlignment>
     <Deterministic>true</Deterministic>
+    <TargetFrameworkProfile />
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:15]: and remove this target frame work reffer decbranch code

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs [hunk 3]
```diff
@@ -1,17 +1,44 @@
...
+                Task<HttpResponseMessage> res = menuItemLibrary.createMenuItem(menuModel);
+                if (res.IsFaulted)
+                {
+                    TempData["message"] = "MenuItem insertion Is UnSuccess";
+                    return RedirectToAction("Create");
+                }
+            }
+            catch (Exception ex)
+            {
+                ViewBag.ErrorMessage = "";
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:36]: add message in viewbag

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj (hunk 1 above)
Comment (praveen957) [reply DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:12]: frame work changed to 4.8

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj (hunk 2 above)
Comment (praveen957) [reply DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:15]: line 15 is removed and saved the code

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs (hunk 3 above)
Comment (praveen957) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:36]: viewbag message  is added and the view bag message is added in view also

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj (hunk 2 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:15]: still it exists line15

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj [hunk 4]
```diff
@@ -21,6 +22,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <CodeAnalysisRuleSet>MixedMinimumRules.ruleset</CodeAnalysisRuleSet>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:25]: REMOVE this line 25

File: DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj [hunk 5]
```diff
@@ -9,9 +9,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>DigiOrderEnterpriseModel</RootNamespace>
     <AssemblyName>DigiOrderEnterpriseModel</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseModel/DigiOrderEnterpriseModel.csproj:12]: version should be as 4.7.2

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs [hunk 6]
```diff
@@ -119,4 +119,4 @@ public async Task<HttpResponseMessage> POSTAsync(string endPoint, string Json)
             return tokenDetails;
         }
     }
-}
+}
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs:122]: add EOF and commit again .library utility should not exist in ur PR

**Pull request merged**